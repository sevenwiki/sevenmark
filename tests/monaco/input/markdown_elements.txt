# Header 1
## Header 2

**Bold text**
*Italic text*
~~Strikethrough~~
__Underline__
^^Superscript^^
__Subscript__

----