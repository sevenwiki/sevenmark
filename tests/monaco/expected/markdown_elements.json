[
  {
    "Header": {
      "content": [
        {
          "Text": {
            "content": "Header 1",
            "location": {
              "end_column": 11,
              "end_line": 1,
              "start_column": 3,
              "start_line": 1
            }
          }
        }
      ],
      "is_folded": false,
      "level": 1,
      "location": {
        "end_column": 1,
        "end_line": 2,
        "start_column": 1,
        "start_line": 1
      }
    }
  },
  {
    "Header": {
      "content": [
        {
          "Text": {
            "content": "Header 2",
            "location": {
              "end_column": 12,
              "end_line": 2,
              "start_column": 4,
              "start_line": 2
            }
          }
        }
      ],
      "is_folded": false,
      "level": 2,
      "location": {
        "end_column": 1,
        "end_line": 3,
        "start_column": 1,
        "start_line": 2
      }
    }
  },
  {
    "Text": {
      "content": "\n",
      "location": {
        "end_column": 1,
        "end_line": 4,
        "start_column": 1,
        "start_line": 3
      }
    }
  },
  {
    "Bold": {
      "content": [
        {
          "Text": {
            "content": "Bold text",
            "location": {
              "end_column": 12,
              "end_line": 4,
              "start_column": 3,
              "start_line": 4
            }
          }
        }
      ],
      "location": {
        "end_column": 14,
        "end_line": 4,
        "start_column": 1,
        "start_line": 4
      }
    }
  },
  {
    "Text": {
      "content": "\n",
      "location": {
        "end_column": 1,
        "end_line": 5,
        "start_column": 14,
        "start_line": 4
      }
    }
  },
  {
    "Italic": {
      "content": [
        {
          "Text": {
            "content": "Italic text",
            "location": {
              "end_column": 13,
              "end_line": 5,
              "start_column": 2,
              "start_line": 5
            }
          }
        }
      ],
      "location": {
        "end_column": 14,
        "end_line": 5,
        "start_column": 1,
        "start_line": 5
      }
    }
  },
  {
    "Text": {
      "content": "\n",
      "location": {
        "end_column": 1,
        "end_line": 6,
        "start_column": 14,
        "start_line": 5
      }
    }
  },
  {
    "Strikethrough": {
      "content": [
        {
          "Text": {
            "content": "Strikethrough",
            "location": {
              "end_column": 16,
              "end_line": 6,
              "start_column": 3,
              "start_line": 6
            }
          }
        }
      ],
      "location": {
        "end_column": 18,
        "end_line": 6,
        "start_column": 1,
        "start_line": 6
      }
    }
  },
  {
    "Text": {
      "content": "\n",
      "location": {
        "end_column": 1,
        "end_line": 7,
        "start_column": 18,
        "start_line": 6
      }
    }
  },
  {
    "Underline": {
      "content": [
        {
          "Text": {
            "content": "Underline",
            "location": {
              "end_column": 12,
              "end_line": 7,
              "start_column": 3,
              "start_line": 7
            }
          }
        }
      ],
      "location": {
        "end_column": 14,
        "end_line": 7,
        "start_column": 1,
        "start_line": 7
      }
    }
  },
  {
    "Text": {
      "content": "\n",
      "location": {
        "end_column": 1,
        "end_line": 8,
        "start_column": 14,
        "start_line": 7
      }
    }
  },
  {
    "Superscript": {
      "content": [
        {
          "Text": {
            "content": "Superscript",
            "location": {
              "end_column": 14,
              "end_line": 8,
              "start_column": 3,
              "start_line": 8
            }
          }
        }
      ],
      "location": {
        "end_column": 16,
        "end_line": 8,
        "start_column": 1,
        "start_line": 8
      }
    }
  },
  {
    "Text": {
      "content": "\n",
      "location": {
        "end_column": 1,
        "end_line": 9,
        "start_column": 16,
        "start_line": 8
      }
    }
  },
  {
    "Underline": {
      "content": [
        {
          "Text": {
            "content": "Subscript",
            "location": {
              "end_column": 12,
              "end_line": 9,
              "start_column": 3,
              "start_line": 9
            }
          }
        }
      ],
      "location": {
        "end_column": 14,
        "end_line": 9,
        "start_column": 1,
        "start_line": 9
      }
    }
  },
  {
    "Text": {
      "content": "\n",
      "location": {
        "end_column": 1,
        "end_line": 10,
        "start_column": 14,
        "start_line": 9
      }
    }
  },
  {
    "Text": {
      "content": "\n",
      "location": {
        "end_column": 1,
        "end_line": 11,
        "start_column": 1,
        "start_line": 10
      }
    }
  },
  "HLine"
]